service/mapper/config.go

```diff
@@ -16,22 +16,22 @@ package mapper
 
 // DefaultConfig has the default values of the config set.
 var DefaultConfig = Config{
-	IndexCommit:       false,
-	IndexHeader:       false,
-	IndexCollections:  false,
-	IndexTransactions: false,
-	IndexEvents:       false,
-	IndexPayloads:     false,
+	IndexCommit:             false,
+	IndexHeader:             false,
+	IndexCollections:        false,
+	IndexTransactions:       false,
+	IndexEvents:             false,
+	IndexPayloads:           false,
```

Comment:
This is no longer `go fmt`ed, is it? Anyway, we need to add `IndexResults` here, and it should be handled separately from transactions.